contacts/tests.py

```diff
@@ -1,3 +1,123 @@
-# from django.test import TestCase
+import pytest
+from users.models import User, Seeker
+from apartments.models import Apartment, City
+from .models import Connection
+from django.db import transaction, IntegrityError
 
-# Create your tests here.
+
+@pytest.fixture
+def new_city():
+    city = City(cityName='Tel Aviv')
+    city.save()
+    return city
+
+
+@pytest.fixture
+def seeker_1(new_city):
+    user1 = create_new_user("t1@m.com", "test1", "test", "1995-05-05", "testing")
+    seeker = create_seeker(user1, new_city, "2020-05-05", 100, 3000, 2, 2, "Hi")
+    seeker.save()
+    return seeker
+
+
+@pytest.fixture
+def seeker_2(new_city):
```

Comment:
You can achieve the same using one fixture Instead of creating two, almost identical ones
for example: 
```
@pytest.fixture
def fixture_with_different_results():
    def _inner_function(par_1, par_2, par_3):
        changing_list = [par_1,par_2,par_3]
        return changing_list

    return _inner_function

def test_the_fixture(fixture_with_different_results):
    first_item = fixture_with_different_results(1,2,3)
    second_item = fixture_with_different_results(4,5,6)
```

first_item will get a list containing [1,2,3]
second_item will get a list containing [4,5,6]

Reply:
It'll also give you more flexibility with the information you want to insert for the test.

Reply:
@Omeramsc Wouldn't that require me to call this fixture and create a seeker inside the test function?
I thought that the main point of a fixture is to have blocks of code happen before the test starts, that's what I got from Edward. For example, if I understand correctly you'd like something like:
```
def test_invalid_users_connection(fixture_with_different_results):
    seeker_1 = fixture_with_different_results(*seeker1 data*)
    seeker_2 = fixture_with_different_results(*seeker2 data*)
    with pytest.raises(ValueError):
        with transaction.atomic():
            Connection(seeker=seeker_1, apartment=seeker_2).save()
```
There's other rows I neglected as well (creating the user, which is part of the seeker data, before calling the fixture), Which will add lines of code and complexity IMO.

Reply:
Shorter isn't always better or less complex; While this feature of fixtures is great, I personally wouldn't create two separate fixture for entries as similar as those. @simzacks @1Oreo  WDYT?

Reply:
I agree with what @Omeramsc said. With the way the project is now (small and in the early days) writing a fixture for every fixture can work, and still be readable and understandable, but in a much bigger application it will again create ambiguity. One of the traits good code needs is to be readable but also understandable and I think what Omer suggested shows someone who reads the code that the fixture is of the same type, just with 2 different users, and also "saves" duplicating code.

Reply:
@Omeramsc I've never seen this and couldn't really find it online, but gave it a shot.
I changed to code, please tell me if this was what you meant, Thanks!

Reply:
This is called factories as fixtures, you can read more about it (and pytest fixtures) here:
https://docs.pytest.org/en/stable/fixture.html#factories-as-fixtures

I left some comments, but it already looks way better. good job. :)

Reply:
@Omeramsc Thanks! Glad to know that this looks better.
I fixed the code due to your comments, please review again and let me know if there's anything else I should change.

Reply:
Also, pytest won't work until #91  & #92  are merged so please ignore it while reviewing.